app.py

```diff
@@ -0,0 +1,151 @@
+import os
+from flask import Flask, render_template, redirect, request, url_for, flash
+from flask_pymongo import PyMongo
+from bson.objectid import ObjectId
+from operator import itemgetter
+from dotenv import load_dotenv #added the .env and .gitignore
+
+load_dotenv()
+
+app = Flask(__name__)
+
+app.config['MONGO_DBNAME'] = 'quotes_library'
+app.config['MONGO_URI'] = os.getenv("MONGO_URI")
+app.secret_key = os.getenv("secret_key")
+
+mongo = PyMongo(app)
+
+@app.route('/')
+
+
+def home():
+    """
+    This function loads the home page
+    """
+    return render_template('home.html', 
+        category=list(mongo.db.categories.find()),
+        sources=list(mongo.db.sources.find()),
+        languages=list(mongo.db.languages.find().limit(50))
+        )
+
+@app.route('/go_home')
+def go_home():
+    return redirect('home')
+
+    
+@app.route('/get_all_languages')
+def get_all_languages():
+    """
+    This function loads the full languages list on the home page
```

Comment:
You can remove "This function" from the comment as there's only one function you could be referring to

Reply:
Removed. And changed the functions routes to make them more readable